docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.3"
+services:
+  echo-client:
+    image: my-repo/echo-client:1.0-SNAPSHOT
+    depends_on:
```

Comment:
The client should not be dependent on the server. If the client is being run and server is not the client should retry and wait while server will be up (Circuit Breaker pattern). Every dependency add new complexity. We should we need to avoid complications much as possible.

Reply:
Fixed